Extension/i18n/esn/package.i18n.json

```diff
@@ -167,16 +167,16 @@
 	"c_cpp.configuration.suggestSnippets.description": "Si se establece en true, el servidor de lenguaje proporciona los fragmentos de código.",
 	"c_cpp.configuration.enhancedColorization.description": "Si se habilita esta opción, el código se colorea de acuerdo con IntelliSense. Esta configuración solo se aplica si intelliSenseEngine se establece en \"Predeterminado\".",
 	"c_cpp.configuration.codeFolding.description": "Si está habilitada, el servidor de lenguaje proporciona intervalos de plegado de código.",
-	"c_cpp.configuration.vcpkg.enabled.markdownDescription": "Habilita los servicios de integración para el [administrador de dependencias de vcpkg](https://aka.ms/vcpkg/).",
+	"c_cpp.configuration.vcpkg.enabled.markdownDescription": "Habilita los servicios de integración para el [administrador de dependencias de vcpkg] (https://aka.ms/vcpkg/).",
```

Comment:
I opened a feature request against VS Code that might help us address this issue: https://github.com/microsoft/vscode/issues/125774